src/hera/shared/_global_config.py

```diff
@@ -40,7 +40,7 @@ class _GlobalConfig:
     # are processed upon accessing. The rest, which use primitive types, such as `str`, can remain public
     _token: Optional[Union[str, TokenGenerator, Callable[[], Optional[str]]]] = None
     """an optional authentication token used by Hera in communicating with the Argo server"""
-
+    _client_certs: Optional[Tuple[str, str]] = None
```

Comment:
What do you think about making this a `Path` object? That way Hera can check that the cert/key files do exist